.github/workflows/03-helm-release.yaml

```diff
@@ -0,0 +1,48 @@
+name: 03-Helm chart release
+
+on:
```

Comment:
We should ignore paths:
```
    paths:
      - 'charts/**.yaml'
      - 'charts/**.tpl'
```
cc: @matanshk

Reply:
@dwertent 
please talk with me, I want to understand why

Reply:
Because we do not want the workflow to run when changes are made to these paths

Reply:
The workflow will be triggered only by the in-cluster component or by triggering it manually, it does not have any other triggers (like push); so it will never run if we are changing something in this paths anyway

Reply:
We cannot trigger this workflow when updating the helm charts manually? What if I add a field?